backend/selectgen.ml

```diff
@@ -156,6 +156,10 @@ let env_close_regions env rs =
   in
   aux None env.regions rs
 
+let select_mutable_flag : Asttypes.mutable_flag -> Mach.mutable_flag = function
```

Comment:
(If we keep the `Mach.mutable_flag` type explicitly
as a non-synonym, maybe move this conversion
function to the `Mach` module - I am afraid no one
would look here for such a function.)

Reply:
Maybe later.  I'd like to get this one merged.